### DISCUSSION TITLE
Define loss function in keras of tensorflow.

### DISCUSSION THREAD

**Discussion by KawaSakiHakuButsu** [deepmodeling/deepmd-kit#1258] [Q&A]
Hello everyone,
If it's possible to define a loss function with both outputs and inputs in keras of tensorflow, how can I do so?
Thanks.

**Comment by AnguseZhang** [comment]
I can't get the relationship between this question and DeePMD-kit.

**Reply by KawaSakiHakuButsu** [reply]
I think that may help to optimize the training process of the DP force field.
